docs/en_US/sqls/functions/mathematical_functions.md

```diff
@@ -255,3 +255,12 @@ degrees(col)
 ```
 
 converted from radians to degrees
+
+## CONV
+
+```text
+conv(col,from_base,to_base)
+```
+
+converts numbers between different number bases. The argument col is interpreted as an integer, but may be specified as
```

Comment:
The return result is a string? Need to clarify that.

Reply:
done